src/ensembl/src/shared/components/panel/Panel.test.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import faker from 'faker';
+
+import Panel, { PanelProps } from './Panel';
+
+const panelContent = <p>{faker.lorem.words()}</p>;
+
+const panelHeader = <p>{faker.lorem.words()}</p>;
+
+const onClose = jest.fn();
+
+const panelClassNames = {
+  panel: faker.lorem.word(),
+  header: faker.lorem.word(),
+  body: faker.lorem.word(),
+  closeButton: faker.lorem.word()
+};
+
+const defaultProps: PanelProps = {
+  header: panelHeader,
+  children: panelContent,
+  classNames: panelClassNames
+};
+
+const renderPanel = (props: PanelProps = defaultProps) => {
+  return <Panel {...props} />;
+};
```

Comment:
We could improvise this with making the `props` parameter optional and include the `defaultProps` in the component as:

const renderPanel = (props?: PanelProps) => {
  return <Panel {...defaultProps} {...props} />;
};

This way we won't need to pass `defaultProps` along with the `props` every time we call this function.

Reply:
> This way we won't need to pass defaultProps along with the props every time we call this function.

I guess that's how it currently works as I have used:
 > props: PanelProps = defaultProps

Reply:
Not really. You now need to always merge `defaultProps` with `props` before passing it to this function.

Reply:
That's updated now